README.md

```diff
@@ -86,11 +86,11 @@ The browser tests use these technologies:
 1. Copy and paste the resulting docker login command into a terminal to authenticate your Docker CLI to the registry. This command provides an authorization token that is valid for the specified registry for 12 hours.
 1. Run `aws ecr describe-repositories`. You should see a list of ECR repositories.
 
-#### Run the scripts
+#### Run the tests
+The following command will (1) start services and applications, (2) run the tests and (3) stop the services and applications.
 
 ```
-$ bin/setup
-$ bin/run
+$ bin/setup && bin/run_tests && bin/stop
```

Comment:
🙋‍♂️ Should we include a CircleCI job to do this as part of this pull request?

Reply:
I'll do that in a separate pull request